src/Versatile_RotaryEncoder.cpp

```diff
@@ -147,9 +151,14 @@ bool Versatile_RotaryEncoder::ReadEncoder() {
             switch (button) {
                 case switchup:
                     encoder = release;
-                    if (handlePressRelease != nullptr) {
```

Comment:
Maybe it needs to check for "(millis() - last_switchup < double_press_duration)" in order to not trigger "handlePressRelease()" when "handleDoublePressRelease()" is intended instead.